nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -107,6 +107,13 @@ $(CsWinRTFilters)
     <WriteLinesToFile File="$(CsWinRTResponseFile)" Lines="$(CsWinRTParams)" Overwrite="true" WriteOnlyWhenDifferent="true" />
     <Message Text="$(CsWinRTCommand)" Importance="$(CsWinRTMessageImportance)" />
     <Exec Command="$(CsWinRTCommand)" />
+    <ItemGroup>
+      <CsWinRTOutputs Include="$(CsWinRTGeneratedFilesDir)/*.cs"/>
+      <UpToDateCheckInput Include="@(CsWinRTInputs)" Set="CsWinRTInputs" />
```

Comment:
In the doc I see `Note that UpToDateCheckOutput exists but is deprecated and only maintained for backwards compatability. Projects should use to UpToDateCheckBuilt instead.`  Thoughts?

Reply:
yes, I saw that too. but the UpToDateCheckBuilt doesn't support grouped dependencies (and even goes on to say that it doesn't compose with sets, implying that UpToDateCheckOuput *is* still supported)

Reply:
the worse case is that we're no better off than before, if VS ever decides to stop supporting (i.e, there's no risk of making things worse)